JavaScript/ApprovedLibraries.md

```diff
@@ -0,0 +1,116 @@
+# Yeti Approved Libraries
+
+## React
+**[react-router](https://github.com/ReactTraining/react-router)**
```

Comment:
Do we also want to include `react-router-redux`?

Reply:
that's a great suggestion! Johnny informed me that `react-router-redux` was actually just deprecated in favor of `connected-react-router` so it will probably be best to add that one instead.

Reply:
e'en better!